lib/evss/configuration.rb

```diff
@@ -45,7 +45,8 @@ def connection
         faraday.use      Faraday::Response::RaiseError
         faraday.response :snakecase, symbolize: false
         faraday.response :json
-        faraday.adapter  :httpclient
+        faraday.use(Common::Client::Middleware::Request::RemoveCookies)
```

Comment:
For cross team stuff please be sure to always include engineers from the team who own the product, neither @omgitsbillryan nor I were originally added as reviewers. 

This now touches all EVSS services that use this base config. What's the testing/release plan for this? I'd recommend at least a smoke test for all EVSS services (not just relying on specs).

Reply:
Agreed re: smoke test. Once final approval is in place let's schedule for testing.